### PR TITLE
Created a new page describing the new notifications interface, added …

### PR DESCRIPTION
**Pull request by chrismcg14**
…warnings for how to turn it off on creating a job and job plugins. Linked the instructions on those pages.

### CODE REVIEW

File: docs/manual/notifications/new-interface.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Notification Interface V2 (After v3.3.4)
```
Comment (fdevans) [inline docs/manual/notifications/new-interface.md]: We don't typically mention version numbers in this context.  We just present it as the current interface so the title would be "Notification Interface" or something that describes this product area.

File: docs/manual/job-plugins.md [hunk 2]
```diff
@@ -49,6 +49,16 @@ Notification plugins allow Rundeck to communicate changes in job execution state
 - [Email](/manual/notifications/email.md)
 - [Webhooks](/manual/notifications/webhooks.md)
 
+:::warning
```
Comment (fdevans) [inline docs/manual/job-plugins.md]: Not convinced these are necessary here.  Maybe just documenting it on the new page you created below that they can revert to the legacy interface but may run into issues.  This new UI was developed due to a scaling bug and is the long term path forward.

**Review by fdevans — CHANGES_REQUESTED** [review]

File: docs/manual/notifications/new-interface.md (hunk 1 above)
Comment (chrismcg14) [reply docs/manual/notifications/new-interface.md]: Updated!

File: docs/manual/job-plugins.md (hunk 2 above)
Comment (chrismcg14) [reply docs/manual/job-plugins.md]: Updated! @fdevans

**Pull request closed without merging**